### PR TITLE
Add support for Iron Irwini

### PR DESCRIPTION
**Pull request by cottsay**
Requires ros-infrastructure/ros2-cookbooks#53

### CODE REVIEW

File: windows_docker_resources/install_ros2_iron.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "ros2_windows": {
+    "download_sources": "false",
+    "vs_version": "buildtools",
+    "qt5": {
+      "installation_method": "offline"
+    },
+    "ros2_ws": "C:/ci",
+    "ros_distro": "rolling",
```
Comment (clalancette) [inline windows_docker_resources/install_ros2_iron.json:9]: Should this be `iron`?

Comment (cottsay) [reply windows_docker_resources/install_ros2_iron.json:9]: yes it should. I copied from rolling and forgot to update the contents.

**Comment by cottsay** [comment]
I'm hoping to test these jobs using the `test_ci_*` jobs on ci.ros2.org prior to merging. I'm still coordinating that.

**Comment by cottsay** [comment]
* Linux [Build Status](http://ci.ros2.org/job/test_ci_linux/125/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/test_ci_linux-aarch64/20/)
* Linux-rhel [Build Status](https://ci.ros2.org/view/All/job/test_ci_linux-rhel/)
* Windows [Build Status](http://ci.ros2.org/job/test_ci_windows/257/)

**Review by clalancette — APPROVED** [review]

**Review by Yadunund — APPROVED** [review]
LGTM!

**Pull request merged**